src/services/user/model/user.schema.js

```diff
@@ -122,6 +122,7 @@ userSchema.index({ firstName: 1, lastName: 1 }); // ok = 5
 userSchema.index({ ldapDn: 1 }); // ?
 userSchema.index({ ldapId: 1 }); // ?
 userSchema.index({ ldapId: 1, schoolId: 1 }); // ?
```

Comment:
Can we remove the line userSchema.index({ ldapId: 1, schoolId: 1 }); // ?  they are mark with question because i do not find any hit or use case in code.

Reply:
It's used by ldap syn now.